src/Nethermind/Nethermind.Merge.Plugin/EngineRpcModule.Paris.cs

```diff
@@ -88,6 +89,11 @@ protected async Task<ResultWrapper<PayloadStatusV1>> NewPayload(IExecutionPayloa
                 using IDisposable region = _gcKeeper.TryStartNoGCRegion();
                 return await _newPayloadV1Handler.HandleAsync(executionPayload);
             }
+            catch (BlockchainException exception)
+            {
+                if (_logger.IsDebug) _logger.Error($"DEBUG/ERROR engine_newPayloadV{version} failed: {exception}");
```

Comment:
Hmmm, not better to keep this error visible by default?

Reply:
Outputs it anyway from JsonRpcError

<img width="896" alt="image" src="https://github.com/user-attachments/assets/54ef85af-7e36-49ae-a760-3f84cdb3f42f" />

Reply:
<img width="793" alt="image" src="https://github.com/user-attachments/assets/2ab39107-dc80-4d7d-bffd-837d6ccd6fab" />